### PR TITLE
Be able to add custom "more items" link for EventListingBlocks

### LINKED ISSUE #52: EventListingBlock custom "More Link"
In the  EventListingBlock you can already define a custom name for the "More Link" now you should also be able to cusomize the url of the link itself. We can achieve this by adding a new ftw.referencewidget where the user can define another >> https://github.com/4teamwork/ftw.referencewidget/#contextsourcebinder

If this link is set this link should be used otherwise the default `eventlistingblock/event_listing` should be used.

### PR DESCRIPTION
**Pull request by tinagerber**
closes #52 

Bildschirmfoto 2019-09-02 um 17 38 59

### CODE REVIEW

File: ftw/events/browser/eventlistingblock.py [hunk 1]
```diff
@@ -30,7 +30,10 @@ def get_block_info(self):
 
         more_items_link_url = ''
         if self.context.show_more_items_link:
-            more_items_link_url = '/'.join([self.context.absolute_url(), 'events'])
+            if self.context.link_to_more_items:
+                more_items_link_url = self.context.link_to_more_items.to_object.absolute_url()
```
Comment (mbaechtold) [inline ftw/events/browser/eventlistingblock.py]: If `self.context.link_to_more_items.to_object` is `None`, this line will raise an exception and the event listing block can't be rendered. We need to make this more robust.

See the test `test_custom_link_to_show_more_items_when_target_is_deleted` which I have pushed as a new commit onto this branch.

File: ftw/events/tests/test_event_listingblock.py [hunk 2]
```diff
@@ -453,6 +459,96 @@ def test_block_renders_link_to_more_items_with_custom_label(self, browser):
...
+                       .titled(u'This is a EventListingBlock'))
+
+        browser.login().visit(page)
+        self.assertEqual(
+            'http://nohost/plone/content-page/this-is-a-eventlistingblock/events',
+            browser.find('More Items').attrib['href']
+        )
+
+        browser.login().visit(block, view='@@edit')
+        browser.fill({'Link to more items': page_to_link})
```
Comment (mbaechtold) [inline ftw/events/tests/test_event_listingblock.py]:
```suggestion
        browser.fill({'Link to more items': page_to_link}).submit()
```

File: ftw/events/tests/test_event_listingblock.py [hunk 3]
```diff
@@ -453,6 +459,96 @@ def test_block_renders_link_to_more_items_with_custom_label(self, browser):
...
+        browser.find_button_by_label('Save').click()
```
Comment (mbaechtold) [inline ftw/events/tests/test_event_listingblock.py]: ✂️

**Comment by tinagerber** [comment]
Ok, I made the changes

**Comment by tinagerber** [comment]
Hups I forgot to update the CHANGELOG. But now it's done.

**Pull request merged**